db/schema.rb

```diff
@@ -35,6 +35,7 @@
     t.string "code"
     t.string "shared_code"
     t.boolean "use_shared_code"
+    t.boolean "watcher_job_active", default: false, null: false
```

Comment:
Is this ended? or left over schema changes?

Reply:
this sneaked in from another feature I had developed locally but is not on master. It's removed now.